hack/setup-minikube-shipwright.sh

```
@@ -0,0 +1,291 @@
+    log "Installed ClusterBuildStrategies:"
+    kubectl get clusterbuildstrategy -o custom-columns=NAME:.metadata.name --no-headers | sed 's/^/  - /'
+}
+
+setup_local_registry() {
+    log "Setting up local registry access"
+
+    # Enable minikube registry addon if not already enabled
+    minikube addons enable registry -p "$CLUSTER_NAME" 2>/dev/null || true
+
+    # Get registry NodePort
+    local REGISTRY_PORT=$(kubectl get svc registry -n kube-system -o jsonpath='{.spec.ports[0].nodePort}' 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)